### PR TITLE
fix(DB/quest): 1582 - Moonglow Vest req. skill

### PR DESCRIPTION
**Pull request by wowquester**
## Changes Proposed:
-  Req. leatherworking skill decreased to 70 (90 -> 70)

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5812

## SOURCE:

https://www.wowhead.com/wotlk/quest=1582/moonglow-vest#comments
(I wouldn't normally agree with wowhead, but the comments are highly upvoted)
Mentioned in this guide: https://youtu.be/WJKHFy9J9GU?t=45

## Tests Performed:

Tested.
WoWScrnShot_070923_215007

## How to Test the Changes:

1. .tele darnassus
2. .setskill 165 70
3. Take the quest

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fixq1582 [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `quest_template_addon` SET `RequiredSkillPoints` = 70 WHERE (`ID` = 1582);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/fixq1582:1]: the name of the file is wrong, please follow this documentation for it https://www.azerothcore.org/wiki/sql-versioning

File: data/sql/updates/pending_db_world/rev_1689021841397941900 [hunk 2]
```diff
@@ -0,0 +1 @@
+UPDATE `quest_template_addon` SET `RequiredSkillPoints` = 70 WHERE (`ID` = 1582);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1689021841397941900:1]: it's missing the .sql in the file name

you probably don't see it properly for a Windows settings that you could remove following this https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-enable-hidden-file-extensions-in-Windows.html

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
I do not know if this requires sniff

**Comment by Gultask** [comment]
It does not. From what I recall nothing in quest_template_addon isn't sniffed. Although Skill Requirement lines/IDs are.

**Pull request merged**